R/cleanup.R

```diff
@@ -85,12 +92,18 @@ orderly_cleanup_data <- function(config) {
 
   csv <- orderly_db("csv", config, FALSE)
   drop_csv <- setdiff(csv$list(), used)
+  msg <- sprintf("Found %s csv %s", length(drop_csv),
```

Comment:
oh man totally forgot about all this old stuff. We should tidy this up at some point...